lib/Target/ARM/ARMPGLTOptimizer.cpp

```diff
@@ -32,6 +32,7 @@ class ARMPGLTOpt : public MachineFunctionPass {
 public:
   static char ID;
   explicit ARMPGLTOpt() : MachineFunctionPass(ID) {}
+  // TODO(yln): why not self-registering
```

Comment:
What do you mean by self-registering? With RegisterPass? We want this run at a very specific point in the pipeline, and I don't think you can register passes in the backend with RegisterPass, can you?